### ISSUE TITLE
ML image to mask with unet

### ISSUE THREAD

**Issue by moellep** [radiasoft/sirepo#5315]
Allow image-to-image h5 input files. A good example would be the oxford pets database:

https://keras.io/examples/vision/oxford_pets_image_segmentation/
